bin/wavTranscription.md

```diff
@@ -0,0 +1,49 @@
+## Transcribing longer audio clips
+
+The Command and GUI tools perform the same operation. Read a wav file of any duration,
+use the WebRTC Voice Activity Detector (VAD) to split it into smaller chunks and finally
+save a consolidated transcript.
+
+### 0. Pre-requisites
+
+```
+sudo apt-get install virtualenv python3-pyqt5 libcanberra-gtk-module
+virtualenv venv
+source venv/bin/activate
+pip3 install webrtcvad; pip3 install deepspeech; pip3 install tensorflow; pip3 install -r requirements.txt
+
+export PATH=$PATH:~/.local/bin/
+```
+
+### 1. Command line tool
+
+The command line tool processes a wav file of any duration and returns a trancript
+which will the saved in the same directory as the input audio file.
+
+The command line tool gives you control over the aggressiveness of the VAD.
+Set the aggressiveness mode, to an integer between 0 and 3.
+0 being the least aggressive about filtering out non-speech, 3 is the most aggressive.
+
+```
+ ~/DeepSpeech$ python3 audioTranscript_cmd.py 1 ../../../audio/original/sample_rec.wav
```

Comment:
nit: It'd be great if you could add argparse handling :)

Reply:
Will do that. It's more readable having it in place :+1: 

I also found a couple of things changed with the 0.2.0 release..
1. `WORD_COUNT_WEIGHT` is no longer used in `enableDecoderWithLM()`
2. `from deepspeech.model import Model` needs to be changed to `from deepspeech import Model, printVersions`

I'll push all of these changes together :)